### DISCUSSION TITLE
Git push ERROR- Help

### DISCUSSION THREAD

**Discussion by Mersag** [neuefische/questions#74] [Web] [answered]
Hallo ,
### Was
ich möchte einen Push ausführen, um einen Pull Request vorzunehmen. 
![image](https://user-images.githubusercontent.com/107836739/185054362-77718c65-61f2-4bae-9677-7e6bc39cb6d5.png)

### Frage
Was muss ich tun oder habe vergessen es zu tun, um einen Push auszuführen?
Ich verstehe die Fehlermeldung nicht?

Was ich ausgeführt habe, ist: 
                                                  - Git add . 
                                                  - git status
                                                  - git commit -m "Nachricht"
                                                  - git push 

Ich danke euch im Voraus für eure Hilfe

**Comment by F-Kirchhoff** [comment, marked as answer]
Du kannst nicht pushen weil ESLint dich daran gehindert hat. weil dein Code anscheinend fehlerhaft ist. ESLint hat deinen Code überprüft und in der Datein `index.js` in Zeile 18 keinen Closing Tag für ein `<div>` Tag gefunden.  Versuche den Closing Tag hinzuzufügen und pushe erneut.

**Reply by Mersag** [reply]
Stimmt . Danke :-)

**Comment by tsimvris** [comment]
Discussion geschlossen , error ist aufgeräumt

**Reply by Mersag** [reply]
Super, Danke für deine Hilfe. Ich habe den  Card Tag in der index.js geändert. :-)
